package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "resumebuilder",
+  "version": "1.0.0",
+  "description": "Tip: Read [General Information.md](\"./General Information.md\") for more info on how to get started and submit.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "npm run start-server & npm run build-watch",
+    "start-dev": "NODE_ENV='development' npm run build-watch & NODE_ENV='development' npm run start-server",
+    "start-server": "nodemon server -e html,js,scss --ignore public --ignore client",
+    "build-watch": "webpack -w"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/w-joy5015/ResumeBuilder.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/w-joy5015/ResumeBuilder/issues"
+  },
+  "homepage": "https://github.com/w-joy5015/ResumeBuilder#readme",
+  "devDependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-react": "^6.24.1",
+    "webpack": "^4.42.0",
+    "webpack-cli": "^3.3.11"
+  },
+  "dependencies": {
+    "express": "^4.17.1",
```

Comment:
I'm curious as to why you're included express as a dependency here? If your using it for a devServer to update your app as you make changes then it should be a devDependency.